object-manipulation-2/src/defaults.js

```diff
@@ -0,0 +1,14 @@
+/* exported defaults */
+
+/*
+  iterate through source object keys
+  if that key not in target keys then add to it
+*/
```

Comment:
Try rewriting this with the tips I gave earlier tonight. Try and make each line of psuedocode --> one line of js